src/control/lib/control/system_test.go

```diff
@@ -546,6 +547,14 @@ func TestControl_SystemQuery(t *testing.T) {
 		fds[i] = system.MustCreateFaultDomainFromString(fdStrs[i])
 	}
 
+	mustNewMember := func(t *testing.T, r system.Rank, uuidStr, uri string, a *net.TCPAddr, s system.MemberState) *system.Member {
```

Comment:
Any reason not to create this as an exported function in the system package? Looks like it's used in a variety of tests.

Reply:
do we really want to export something in a production package that's only used in test? I fixed the NewMember constructor to return an error if UUID doesn't parse but now I think as it's only used in test I'm going to move it to system/mocks.go and rename it MockMemberFullSpec(. see what you think about the changes.